src/mmw/js/src/compare/views.js

```diff
@@ -101,10 +114,28 @@ var CompareScenarioView = Marionette.LayoutView.extend({
         modificationsRegion: '.modifications-region'
     },
 
+    initialize: function(options) {
+        this.projectModel = options.projectModel;
+    },
+
     onShow: function() {
-        this.mapRegion.show(new CompareMapView({
-            model: this.model
-        }));
+        this.mapModel = new coreModels.MapModel({});
+        this.mapModel.set('areaOfInterest', this.projectModel.get('area_of_interest'));
+        this.mapView = new coreViews.MapView({
+            model: this.mapModel,
+            el: $(this.el).find('.map-container').get(),
```

Comment:
To return a DOM element as opposed to a jQuery selector, since that's what el is supposed to be. Maybe I could just do `$el: $(this.el).find('.map-container')`?